src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    int calculateValueByOperation(FruitTransaction model);

```

Comment:
Після рефокторингу з новими сервісами хендлеру не обов'язково повертати дані
```suggestion
    void calculateValueByOperation(FruitTransaction model);
```